docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/create-space-syntax.md

```diff
@@ -49,3 +49,26 @@ nebula> CREATE SPACE my_space_2(partition_num=10); -- create space with default
 nebula> CREATE SPACE my_space_3(replica_factor=1); -- create space with default partition number
 nebula> CREATE SPACE my_space_4(partition_num=10, replica_factor=1);
 ```
+
+## Checking Partition Distribution
+
+On some large clusters, due to the different startup times, the partition distribution may be unbalanced. You can check the machine and distribution by the following command (SHOW HOSTS).
```

Comment:
times => time

Reply:
clusters

Reply:
> clusters

Time as a countable and uncountable noun https://www.englishgrammar.org/time-countable-uncountable-noun/?__cf_chl_captcha_tk__=a5c144f70c7da337b7793fbfd18d1a39c4df0342-1587965216-0-AaT310lSBgK6SgXwyctwsDqExGZSTYy540K92AlzR7siHrJ5LZKpYaMWIdxtoaaAlGuybnEhIg18glgCgsDikHhT3XqlHycKOJgfzbR8I_naW1jMNdTZr7wzutYkUbO0EKJJeSMjRxzmrgX_6nyjYCtqvMexeLRQxx0jLW8qZRHhqYQLjlaQ1hb1-xAtb1EPJYm86NBhEV9qhcukP_wu9gP3-IbW7_jRD0wJCruNibbP0ySUPsrosY1F6Ki8v-aYHSXn4CFvLFVoU6ZcgyX_MLC4WG4zJKs0y0dOQHbZ4e83fgnG52cUjrO_mR8SfE_sUfT_F1fePT2g9i9XrVgxGNOvnOC9-cyeRWYCJZvaLBcD